support-lambdas/stripe-intent/cfn.yaml

```diff
@@ -61,8 +65,15 @@ Resources:
       Properties:
         AlarmActions:
           - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:reader-revenue-dev
-        AlarmName: StripeIntentLambdaAlarm
-        AlarmDescription: Alarm if StripeIntentLambda errors too many times
+        AlarmName: !Join
+            - ' '
+            - - !FindInMap [ Constants, Alarm, Urgent ]
+              - 'PROD'
+              - 'Stripe create setup intent Lambda failing'
+        AlarmDescription: !Join
+            - ' '
+            - - 'Impact - Credit card checkout will not work for subscriptions or recurring contributions.'
+              - !FindInMap [ Constants, Alarm, Process ]
         Namespace: AWS/Lambda
         MetricName: Errors
```

Comment:
if you are concerned about missing some types of failure e.g. stripe is down, as mentioned in chat, feel free to add an alarm on a custom metric in support-frontend.  There is an AwsCloudwatchMetricPut file I think which should let you put arbitrary metrics.
But I suspect we would return support-frontend 5xx errors if anything went wrong anyway.

Reply:
good point, I will remove it and it will default to missing